ui/src/components/card/CardHeader.js

```diff
@@ -6,7 +6,7 @@ import {
   EuiIcon,
 } from "@elastic/eui";
 
-export const ExperimentCardHeader = ({ onDelete, dragHandleProps }) => (
+export const CardHeader = ({ onDelete, dragHandleProps }) => (
```

Comment:
Refactored `ExperimentCardHeader` to `CardHeader` for use across Experiment Engine and Traffic Rules Drag&Drop components.

Reply:
Shall we name this `DraggableCardHeader` instead to more accurately reflect its use?